src/datadog/string_util.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+#include "propagation_style.h"
+
+namespace datadog {
+namespace tracing {
+
+// Converts a boolean value to a string
+// The resulted string is either `true` or `false`
+std::string to_string(bool b);
```

Comment:
Since you don't refer to the variable name in the contract, then it's safe to omit it, especially here where the interpretation of the parameter is obvious.

Alternatively, you could go full pedant and write it this way:
```c++
// Return a string representation of the specified boolean `value`.
// The result is "true" for `true` and "false" for `false`.
std::string to_string(bool value);
```